docs/workflows/agile-development-process.md

```diff
@@ -0,0 +1,93 @@
+# Using Sigrid as part of an agile development process
+
+Sigrid supports all development process methodologies. That said, most organizations are either already using an [agile development process](https://en.wikipedia.org/wiki/Agile_software_development), or are currently in the process of migrating towards such a way of working. Working agile is therefore both the most mainstream and most desirable way of doing software development. 
+
+This page explains how Sigrid fits into a [Scrum](https://en.wikipedia.org/wiki/Scrum_(software_development) development process, which is the most popular and widely used version flavor of agile. Many of thes best practices outlined in this article can also be used when using other agile project management methodologies, such as [Kanban](https://en.wikipedia.org/wiki/Kanban).
+
+## In an agile process, different roles still have different needs
+
+In Scrum, [developers](developer.html) and [product owners](product-owner.html) need to collaborate to prioritize the sprint backlog. This can be challenging because priorities can be slightly different, or at least *perceived* differently.
+
+<img src="../images/agile-prioritization.png" width="500" />
+
+- **Product owners** generally see prioritization from the customer perspective: adding functionality tends to be seen something that brings most customer value, and is therefore top of mind. 
+- **Developers** generally put additional emphasis on non-functionals, for the obvious reason they need to work with the code on a daily basis. 
+- **Architects** have an interesting role in this process, as they are not "officially" part of the team. However, in many organization architects do in fact have [hands-on involvement with achieving technical goals](architect.html). This can be true even if nobody has "architect" as a job title! This is often called "emerging architect", where a senior engineer (or sometimes even the product owner!) takes care of the duties normally associated with an architect.
```

Comment:
Maybe we can say "in many organizations, ..."

Reply:
OK, in general I tend to use the word "however" ludicrously often, so that also avoids some repetition.